src/containers/SignUp/ThankYou/ThankYou.js

```diff
@@ -59,8 +72,8 @@ class ThankYou extends Component {
                                 </Typography.Paragraph>
                                 <img src={capitol} className={styles.Photo} alt="U.S. Capitol Building" />
                                 <div className={styles.CantWait}>
-                                    <Typography.Title level={4}>Can't wait to start calling?</Typography.Title>
-                                    <Button block onClick={this.handleMakeCallClick} type="primary">Make your first call now!</Button>
+                                    <Typography.Title level={4}>{followUpAction.title}</Typography.Title>
+                                    <Button block onClick={() => followUpAction.handler()} type="primary">{followUpAction.description}</Button>
```

Comment:
Small Nit: You can just pass the function bare if there's nothing special getting passed to it: 

```suggestion
                                    <Button block onClick={followUpAction.handler} type="primary">
                                        {followUpAction.description}
                                    </Button>
```